readme.md

```diff
@@ -1,115 +1,96 @@
 
-## Additional Utilities
-In addition to test file creation features, the repository contains utility functions that work with existing test file data. These functions assist in creating test files with further customization, such as files with a large number of rows or files as part of a new institution. The functions are located in [`python/utils.py`](https://github.com/cfpb/hmda-test-files/blob/master/python/utils.py). 
+For 2019: 
+1. Run `cd 2019/python` to navigate to the python directory. 
+2. Run `python3 generate_2019_error_files.py`
+3. The error files files for testing syntax, validity, and quality edit test files should appear in the following diretories:
+	- Syntax: `2019/edits_files/test_files/{Bank Name}/syntax`
+	- Validity: `2019/edits_files/test_files/{Bank Name}/validity`
+	- Quality: `2019/edits_files/test_files/{Bank Name}/quality`
 
-These functions include:
-- `write_file(path, ts_input, lar_input, name="test_file.txt")` 
-	Writes TS and LAR dataframes to a HMDA submission test file with a specified file path and directory. 
+Note: the filepath and filename for test files can be adjusted in the [2019 Test Filepaths](https://github.com/cfpb/hmda-test-files/tree/master/2019/configurations/test_filepaths.yaml) YAML configuration file under the keys, `syntax_filepath`, `validity_filepath`, and `quality_filepath`.  
 
-- `read_data_frames(ts_df, lar_df)` 
-	Reads in a HMDA submission test file and outputs a TS and LAR dataframe. 
+For 2018: 
+1. Run `cd 2018/python` to navigate to the python directory. 
+2. Run `python3 generate_2018_error_files.py`
+3. The error files files for testing syntax, validity, and quality edit test files should appear in the following diretories:
+	- Syntax: `2018/edits_files/test_files/{Bank Name}/syntax`
+	- Validity: `2018/edits_files/test_files/{Bank Name}/validity`
+	- Quality: `2018/edits_files/test_files/{Bank Name}/quality`
 
-- `unique_uli(new_lar_df, lei)`
-	Writes a set of unique ULI’s with check digits to a LAR dataframe given an LEI input. 
+Note: the filepath and filename for test files can be adjusted in the [2018 Test Filepaths](https://github.com/cfpb/hmda-test-files/tree/master/2018/configurations/test_filepaths.yaml) YAML configuration file under the keys, `syntax_filepath`, `validity_filepath`, and `quality_filepath`. 
 
-- `new_lar_rows(row_count, lar_df, ts_df)` 
-	Duplicates or retracts rows in a LAR dataframe to produce a file with a certain number of rows. Provides capabilities for creating LAR dataframes with large row counts. 
+## Generating Large Files 
+Often, large files would need to be generated to support load testing. The repository allows users to creates a test file based on existing test file data to create a new file with a larger number of rows.
```

Comment:
Change to `Large files are used during load testing`

Reply:
Change from:
 `The repository allows users to creates a test file based on existing test file data to create a new file with a larger number of rows.`

to: `Large test files are created using a clean test file base and the user-specified row count in the [large file configuration](link to large file configuration).`